apps/tangle-dapp/app/restake/page.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
not sure why we need 'use client' here

Reply:
Great catch! 👍 Previously, this component included client data fetching, so I incorporated the `'use client'` directive. However, we no longer require it.